src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/model/JtsMvt.java

```diff
@@ -0,0 +1,81 @@
+package com.wdtinc.mapbox_vector_tile.adapt.jts.model;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * JTS model of a Mapbox Vector Tile.
+ */
+public class JtsMvt {
+
+    /** Map layers by name */
+    private final Map<String, JtsLayer> layersByName;
+
+    /**
+     * Create an empty MVT.
+     */
+    public JtsMvt() {
+        this(Collections.emptyList());
+    }
+
+    /**
+     * Create a MVT with the provided layers.
+     *
+     * @param layers
+     */
+    public JtsMvt(Collection<JtsLayer> layers) {
```

Comment:
no strong feelings here.  We notice that the previous interface supported `Layer... layers` and therefore could be constructed with a single layer / minimal syntax.